### DISCUSSION TITLE
Final Release of 2.2.0

### DISCUSSION THREAD

**Discussion by bfwissel** [dotnet/yarp#2577] [Q&A] [answered]
Is there at least a tentative release date for v2.2.0?  We've been holding up our upgrade to .NET 8 due to issues with v2.1.0.

**Comment by MihaZupan** [comment]
Have you tried using the [v2.2.0-preview1](https://github.com/microsoft/reverse-proxy/releases/tag/v2.2.0-preview1)?
Does it resolve your issues with 2.1.0? If not, could you describe said issues?

We haven't heard any feedback about issues with the 2.2.0 preview so far, so an eventual 2.2 release will be barely different.

**Reply by karelz** [reply, marked as answer]
We plan to release 2.2.0 in roughly 1 week.

**Comment by bfwissel** [comment]
Thank you for the update!
